generation/hermetic_library_generation.sh

```diff
@@ -75,6 +75,13 @@ message="chore: generate libraries at $(date)"
 
 git checkout "${target_branch}"
 git checkout "${current_branch}"
+# if the last commit doesn't contain changes to generation configuration,
```

Comment:
In addition to or in place of the comments here, can we add a description to the hermetic_library_generation workflow to describe the behavior?

Reply:
I added a comment at the beginning of this script.

Reply:
Actually I mean the workflow `hermetic_library_generation.yaml` not the script, as that is usually the entry point people start troubleshooting things. But this change looks good too.